.github/workflows/pathogen-repo-build.yaml

```diff
@@ -28,6 +28,14 @@ on:
           Nextstrain runtime under which to run the build.
           Currently only supports docker, conda, and aws-batch.
           Defaults to "docker".
+
+          The aws-batch runtime requires the secrets:
+
+          - AWS_ACCESS_KEY_ID
+          - AWS_SECRET_ACCESS_KEY
+
+          They must be defined in the repo's Actions secrets and passed to this
+          workflow with `secrets: inherit`.
```

Comment:
_non-blocking_

We can loosen this restriction in the future by having this workflow automatically arrange for short-lived session keys using the OIDC token exchange.

Reply:
Ah, we would still have to require an input from the caller workflow for the role-to-assume, but that can just be a plain string input rather than a secret or environment variable.

Reply:
I was thinking we'd have a single role used by all our AWS Batch builds, so the workflow here could assume it without further input.